src/rules/__tests__/prefer-hooks-on-top.test.ts

```diff
@@ -0,0 +1,200 @@
+import { TSESLint } from '@typescript-eslint/experimental-utils';
+import rule from '../prefer-hooks-on-top';
+
+const ruleTester = new TSESLint.RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+  },
+});
+
+ruleTester.run('basic describe block', rule, {
+  valid: [
+    [
```

Comment:
I tend to prefer using template strings, as they make it easier to modify things.

My main gripe however is that we've not got consistency across our tests 😭

Reply:
Yes, a common standard would be good. I didn't know some tests used template strings.

This is from `no-test-return-statement`, it would seem that there is extra indentation added to it. Is that ok? because that would require modifying the location of the error as well
```js
    `
    it("one", function () {
      expect(1).toBe(1);
    });
    `
  ]
```

Reply:
And there you've hit the nail on moving to templates: to what level do we indent?

The snippet you've pasted is what I'd go w/ - the indentation is fine, as it's still completely valid test wise. It's either recalculate or call` .trim`.

Either way, the test still represent the same thing 🙂